mat64/dense.go

```diff
@@ -257,7 +257,7 @@ func (m *Dense) SetRow(i int, src []float64) int {
 	return min(len(src), m.mat.Cols)
 }
 
-// RowView returns a Vector reflecting row i, backed by the matrix data.
+// RowView returns a Vector reflecting row i transposed, backed by the matrix data.
```

Comment:
What I don't like about your current wording is that it implies something special is happening, which it isn't. The method is doing the logical thing. The documentation just needs a nudge to remind the user that the Vector=column invariant isn't being broken.

Alternate idea:
"RowView returns row i of the matrix data represented as a column vector"